src/main/java/frc/robot/commands/shooter/AdvancedSpinningShot.java

```diff
@@ -45,7 +45,7 @@ public void end(boolean interrupted) {
 
     @Override
     public boolean isFinished() {
-        return alignable == null || alignableSupplier.get() == null || !alignable.equals(alignableSupplier.get());
+        return false;
```

Comment:
why remove the termination condition?

Reply:
How it was implemented when we went to WPI was that the advanced shot was independent of the alingable's supplier current value, so they did not want the command to end when they released the trigger to align to the speaker. However now that the advanced shot is tied to current alignable being held I could add the condition back.